config/cdb.conf

```diff
@@ -0,0 +1,17 @@
+# if slab profile is specified, then the profile wil be explicitly set
+# otherwise, slab profile will be generated by using growth factor, slab size,
+# and item min/max size
+# For example, you can specify all item sizes allowed with slab_profile:
+# slab_profile: 1024 2048 4096 8192 16384 32768 65536 131072 262144 524288
+
+debug_log_level: 4
+debug_log_file: cdb.log
+debug_log_nbuf: 1048576
+
+klog_file: cdb.cmd
+klog_backup: cdb.cmd.old
+klog_sample: 100
+klog_max: 1073741824
+
+# TODO: this is _obviously_ temporary
+# cdb_file_path: /Users/jsimms/git/tub/cdb.rs/dict.cdb
```

Comment:
I wasn't sure how the path to the cdb file should be specified, i.e. config file or command-line arg